audio-worklet/noisegate-test.html

```diff
@@ -339,7 +338,11 @@
         const length = 100;
         let envelope = Array.apply(null, {length}).map((data, index) => index);
         const transitionIndex = length / 2;
-        const threshold = length / 2;
+
+        // Compute weights uses a sine normalized decibel scale.
+        const threshold = NoiseGate.toDecibel_(
+            Math.sqrt(envelope[length / 2]) * Math.sqrt(2));
```

Comment:
"length/2" should be "transitionIndex"? Or maybe they're really different things that just happen to have the same value?